docs/eventing/broker/README.md

```diff
@@ -36,6 +36,50 @@ metadata:
 EOF
 ```
 
+## Explicit Broker configuration
+
+You can of course create a Broker without relying on the defaults (as described
+above) by specifying the `BrokerClass` by using an annotation to specify which
+Broker implementation you'd like to use:
```

Comment:
```suggestion
You can configure Knative Eventing so that when you create a broker, it uses a different type of broker than the default Knative channel-based broker.

To configure a different broker type, or *class*, you must modify the `eventing.knative.dev/broker.class` annotation and `spec.config` for the Broker object.

`MTChannelBasedBroker` is the broker class default.
```

Reply:
kafka, gcp, hopefully soon rabbit too.

Reply:
Can you add maybe an example here that doesn't use MT if that's the default, i.e. change the example to Kafka or something? Can we also list all the available options?

Q: Are there examples of different ConfigMaps for all of these, can we change the `spec.config` in the example too so that it points to a different ConfigMap, or is that default in the sense that you have to configure it first to define the default channel type, rather than default meaning InMemoryChannel or something?